frontend/src/components/shared/header/nav/index.tsx

```diff
@@ -8,7 +8,7 @@ interface INavProps {
 class Nav extends React.Component <INavProps, {}>{
   render() {
     const addClass = this.props.addClass || '';
-    const logo = this.props.logo || "/assets/shared/dps-logo-white.svg";
+    const logo = this.props.logo || '/assets/shared/dps-logo-white.svg';
```

Comment:
Is there a specific reason, why you changed the quotes?

Reply:
I think semantically there is no difference, just because everywhere else in js we use '' and "" in html-like syntax

Reply:
Is there a coding style guide coming up? ;)

Reply:
Gatsby will include prettier 😎